### ISSUE TITLE
Feature Request - Control Incoming Audio in Group Call

### ISSUE THREAD

**Issue by sagarjoshi-sct** [Azure/Communication#385]
**Is your feature request related to a problem? Please describe.**
I'm writing an app where a user can participate in 1 or more ACS Group audio-video calls simultaneously. They should be able to control incoming audio in a given call, without affecting audio from other calls or applications.

**Describe the solution you'd like**
User should be able to 
- Block/Unblock collective incoming audio from a specific call
- Control volume of collective incoming audio from a specific call

**Describe alternatives you've considered**
Couldn't find any API that will let me do this. [This post](https://docs.microsoft.com/en-us/answers/questions/560193/azure-communication-service-audio-control.html?childToView=565704#answer-565704) mentions that the feature is not yet supported.

**Note**
This is a feature-request and not a bug report, but I'm unable to edit the labels.

**Comment by mariusu-msft** [comment]
HI @sagarjoshi-sct,

Thanks for your report. Could you clarify which SDK you are consuming?

Thanks,
Marius

**Comment by sagarjoshi-sct** [comment]
Hi @mariusu-msft,

I'm using ACS Calling JS SDK (`@azure/communication-calling": "^1.1.0"`)

**Comment by ProbablePrime** [comment]
Move to: https://feedback.azure.com/d365community/idea/0919ff5b-1353-ec11-8f8e-000d3a987052

**Comment by fizampou** [comment]
Hi @sagarjoshi-sct,

You can achieve your asks with existing published versions, please upgrade to a newer version, preferably the [latest one](https://www.npmjs.com/package/@azure/communication-calling/v/1.8.1-beta.1).

On 1.6.0 alpha version we introduce media access APIs where application can send/receive custom audio streams. Here is an example [PR](https://github.com/Azure-Samples/communication-services-web-calling-tutorial/pull/58/files). With the same release we exposed another API where application can mute/unmute incoming audio example [PR](https://github.com/Azure-Samples/communication-services-web-calling-tutorial/pull/59/files#diff-ad3183cb1a19f87fea5def6b9a86a8d4876df0fa3de204768c2b7ca75f5ec297).

To sum up, you can mute incoming audio on one call and get the raw stream out it and pipe it to your own audio element and on that you can control the volume or any of the available properties.
